src/receivers/SocketModeReceiver.ts

```diff
@@ -147,16 +148,9 @@ export default class SocketModeReceiver implements Receiver {
     this.app = app;
   }
 
-  public start(): Promise<void> {
-    return new Promise((resolve, reject) => {
-      try {
-        // start socket mode client
-        this.client.start();
-        resolve();
-      } catch (error) {
-        reject(error);
-      }
-    });
+  public start(): Promise<void | WebAPICallResult> {
+    // start socket mode client
+    return this.client.start();
```

Comment:
Nice use of the OR instead of updating it to only `WebAPICallResult`.

Do we need to add any test coverage for this update?

Reply:
Do we need test coverage? Yes
Do we have tests for SocketModeReceiver yet?....no :(